From khali@linux-fr.org Mon Oct 17 13:53:08 2005
Date: Mon, 17 Oct 2005 22:51:37 +0200
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Subject: [PATCH 02/14] Owner field additions to many i2c drivers, 2 of 5
Message-Id: <20051017225137.75a0adfb.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-owner-field-02-struct-device-driver.patch

From: Laurent Riffard <laurent.riffard@free.fr>

This patch updates the .owner field for various struct xxxx_driver variables,
other than pci_driver.

Signed-off-by: Laurent Riffard <laurent.riffard@free.fr>
Signed-off-by: Jean Delvare <khali@linux-fr>
Acked-by: Mark A. Greer <mgreer@mvista.com>
Acked-by: Ben Dooks <ben-linux@fluff.org>
Acked-by: Deepak Saxena <dsaxena@mvista.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/i2c/busses/i2c-iop3xx.c  |    1 +
 drivers/i2c/busses/i2c-ixp2000.c |    1 +
 drivers/i2c/busses/i2c-ixp4xx.c  |    1 +
 drivers/i2c/busses/i2c-mpc.c     |    1 +
 drivers/i2c/busses/i2c-mv64xxx.c |    1 +
 drivers/i2c/busses/i2c-s3c2410.c |    2 ++
 drivers/i2c/chips/isp1301_omap.c |    1 +
 7 files changed, 8 insertions(+)

--- gregkh-2.6.orig/drivers/i2c/busses/i2c-iop3xx.c
+++ gregkh-2.6/drivers/i2c/busses/i2c-iop3xx.c
@@ -525,6 +525,7 @@ out:
 
 
 static struct device_driver iop3xx_i2c_driver = {
+	.owner		= THIS_MODULE,
 	.name		= "IOP3xx-I2C",
 	.bus		= &platform_bus_type,
 	.probe		= iop3xx_i2c_probe,
--- gregkh-2.6.orig/drivers/i2c/busses/i2c-ixp2000.c
+++ gregkh-2.6/drivers/i2c/busses/i2c-ixp2000.c
@@ -146,6 +146,7 @@ static int ixp2000_i2c_probe(struct devi
 }
 
 static struct device_driver ixp2000_i2c_driver = {
+	.owner		= THIS_MODULE,
 	.name		= "IXP2000-I2C",
 	.bus		= &platform_bus_type,
 	.probe		= ixp2000_i2c_probe,
--- gregkh-2.6.orig/drivers/i2c/busses/i2c-ixp4xx.c
+++ gregkh-2.6/drivers/i2c/busses/i2c-ixp4xx.c
@@ -155,6 +155,7 @@ static int ixp4xx_i2c_probe(struct devic
 }
 
 static struct device_driver ixp4xx_i2c_driver = {
+	.owner		= THIS_MODULE,
 	.name		= "IXP4XX-I2C",
 	.bus		= &platform_bus_type,
 	.probe		= ixp4xx_i2c_probe,
--- gregkh-2.6.orig/drivers/i2c/busses/i2c-mpc.c
+++ gregkh-2.6/drivers/i2c/busses/i2c-mpc.c
@@ -361,6 +361,7 @@ static int fsl_i2c_remove(struct device 
 
 /* Structure for a device driver */
 static struct device_driver fsl_i2c_driver = {
+	.owner = THIS_MODULE,
 	.name = "fsl-i2c",
 	.bus = &platform_bus_type,
 	.probe = fsl_i2c_probe,
--- gregkh-2.6.orig/drivers/i2c/busses/i2c-mv64xxx.c
+++ gregkh-2.6/drivers/i2c/busses/i2c-mv64xxx.c
@@ -570,6 +570,7 @@ mv64xxx_i2c_remove(struct device *dev)
 }
 
 static struct device_driver mv64xxx_i2c_driver = {
+	.owner	= THIS_MODULE,
 	.name	= MV64XXX_I2C_CTLR_NAME,
 	.bus	= &platform_bus_type,
 	.probe	= mv64xxx_i2c_probe,
--- gregkh-2.6.orig/drivers/i2c/busses/i2c-s3c2410.c
+++ gregkh-2.6/drivers/i2c/busses/i2c-s3c2410.c
@@ -896,6 +896,7 @@ static int s3c24xx_i2c_resume(struct dev
 /* device driver for platform bus bits */
 
 static struct device_driver s3c2410_i2c_driver = {
+	.owner		= THIS_MODULE,
 	.name		= "s3c2410-i2c",
 	.bus		= &platform_bus_type,
 	.probe		= s3c24xx_i2c_probe,
@@ -904,6 +905,7 @@ static struct device_driver s3c2410_i2c_
 };
 
 static struct device_driver s3c2440_i2c_driver = {
+	.owner		= THIS_MODULE,
 	.name		= "s3c2440-i2c",
 	.bus		= &platform_bus_type,
 	.probe		= s3c24xx_i2c_probe,
--- gregkh-2.6.orig/drivers/i2c/chips/isp1301_omap.c
+++ gregkh-2.6/drivers/i2c/chips/isp1301_omap.c
@@ -888,6 +888,7 @@ static int otg_remove(struct device *dev
 }
 
 struct device_driver omap_otg_driver = {
+	.owner		= THIS_MODULE,
 	.name		= "omap_otg",
 	.bus		= &platform_bus_type,
 	.probe		= otg_probe,
